language-design.md

```diff
@@ -102,7 +321,64 @@ you do so? If not, why not?
 
 **Response**
 
-
+The two experiences of natural languages are definitely at odds with each other.
+If you create too natural of a language then syntax becomes too cluttered and
+harder to read. However, a language lacking much resemblance to the English
+language might as well be greek symbols. (Anthony speaks greek so maybe he would
+be able to read it).
+
+> As a side note, are there localized versions of programming languages that
+> don't use English or have multiple options?
+
+We both decided that some aspect of natural language should be included in the
+design of a DSL. The main idea is there needs to be a good balance. When
+designing a Domain Specific Language, fluency and naturalness are extremely
+important to not only get users using the language but also create an easy way
+to interact with the code. There
+should definitely be a strict syntax that doesnt allow for too many words to be
+used sequentially. Essentially, it doesnt need to read like Shakespeare (or a
+good author) but it definitely shouldn't read like Perl or AWK. No matter what,
+it needs to be clear to the user that to maintain code readability, grammar
+may be broken sometimes.
+
+There are benefits of non-natural languages though. Even though there might be a
+higher learning curve with one, once you learn to think in a certain
+language/style it seems useful and more efficient. Even Fowler said a language
+should still feel like a programming language not just English.
+
+To complicate the issue even more, both natural languages and non-natural
+languages suffer from syntax obstacles. One issue we have with Perl is that the
+following piece of code exists and is expected to be readable:
+
+```
+print((($line=join("",<>))=~s/.*\n/index($`,$&)>=$[?"":$&/ge&&$line));
+```
+
+I am sure with enough time someone could figure out what this says but honestly
+this shouldnt exist. The Natural-language equivalent is adding in so many filler
+words to create grammar that the code is cluttered and you cant get a good idea
```

Comment:
I think the problem with the natural language approach is that people tend to think of it as having to be literal natural language - I think a better approach is to create a limited grammar that still resembles natural language, and uses names that more closely align with natural language (which I think your second example does fairly well)